### ISSUE TITLE
learning rate for SGD and ADAM 

### ISSUE THREAD

**Issue by avqmar** [AlonLahav/MeshWalker#10]
I was trying out SGD, and found that params.learning_rate[0] is not defined!

**Comment by AlonLahav** [comment]
Hi,
Since you don't use `cycle` as the `optimizer_type`, you need to define the parameters needed.
In your case, you can try `params.learning_rate = [0.0005]` or other number (I don't remember the optimal number in this case).
